internal/app/decision_maker.go

```diff
@@ -43,9 +43,10 @@ func buildState(s *state) *currentState {
 			}()
 			if flags.contextOverride == "" {
 				r.HelmsmanContext = getReleaseContext(r.Name, r.Namespace)
```

Comment:
This will print the line for every app change which is unnecessary and will be too noisy